### PR TITLE
Add InfoBadge

### PR DESCRIPTION
**Pull request by ghost1372**
## Description

Added InfoBadge Example

## Motivation and Context

InfoBadge is missing in Samples and winui-gallery

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: WinUIGallery/ControlPages/InfoBadgePage.xaml [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        </local:ControlExample>
+
+        <local:ControlExample x:Name="Example3" HeaderText="Placing an InfoBadge Inside Another Control" HorizontalContentAlignment="Stretch" >
+            <local:ControlExample.Example>
+                <Button Padding="0" Width="200" Height="60" HorizontalAlignment="Center" HorizontalContentAlignment="Stretch" VerticalContentAlignment="Stretch">
+                    <Grid HorizontalAlignment="Stretch" VerticalAlignment="Stretch" Width="Auto" Height="Auto">
+                        <SymbolIcon Symbol="Sync" HorizontalAlignment="Center"/>
+                        <InfoBadge Background="#C42B1C" HorizontalAlignment="Right" VerticalAlignment="Top">
+                            <InfoBadge.IconSource>
+                                <FontIconSource FontFamily="Segoe MDL2 Assets" Glyph="&#xF13C;"/>
```
Comment (Jay-o-Way) [inline WinUIGallery/ControlPages/InfoBadgePage.xaml:107]: Font definitions like this should never be hardcoded. It's also old. Change it to StaticResource SymbolThemeIconFont

**Review by Jay-o-Way — CHANGES_REQUESTED** [review]
Please use StaticResource

File: WinUIGallery/ControlPages/InfoBadgePage.xaml (hunk 1 above)
Comment (ghost1372) [reply WinUIGallery/ControlPages/InfoBadgePage.xaml]: @Jay-o-Way done

File: WinUIGallery/DataModel/ControlInfoData.json [hunk 2]
```diff
@@ -1279,6 +1279,32 @@
...
+          "ImagePath": "ms-appx:///Assets/ControlImages/InfoBadge.png",
+          "ImageIconPath": "ms-appx:///Assets/ControlIcons/MenuBarIcon.png",
+          "Description": "Badging is a non-intrusive and intuitive way to display notifications or bring focus to an area within an app - whether that be for notifications, indicating new content, or showing an alert.",
+          "Content": "<p>Look at the <i>InfoBadgePage.xaml</i> file in Visual Studio to see the full code for this page.</p>",
+          "IsNew": true,
+          "IsUpdated": false,
+          "Docs": [
+            {
+              "Title": "InfoBadge - API",
+              "Uri": "https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.infobadge"
```
Comment (Jay-o-Way) [inline WinUIGallery/DataModel/ControlInfoData.json:1296]: Would you be so kind to remove the language tag? `en-us/`

File: WinUIGallery/DataModel/ControlInfoData.json [hunk 3]
```diff
@@ -1279,6 +1279,32 @@
...
+            },
+            {
+              "Title": "Guidelines",
+              "Uri": "https://learn.microsoft.com/en-us/windows/apps/design/controls/info-badge"
```
Comment (Jay-o-Way) [inline WinUIGallery/DataModel/ControlInfoData.json:1300]: Same

**Review by Jay-o-Way — COMMENTED** [review]
Lang tags

File: WinUIGallery/DataModel/ControlInfoData.json (hunk 3 above)
Comment (ghost1372) [reply WinUIGallery/DataModel/ControlInfoData.json]: @Jay-o-Way Done

File: WinUIGallery/ControlPages/InfoBadgePage.xaml [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+        </local:ControlExample>
+
+        <local:ControlExample x:Name="Example3" HeaderText="Placing an InfoBadge Inside Another Control" HorizontalContentAlignment="Stretch" >
+            <local:ControlExample.Example>
+                <Button Padding="0" Width="200" Height="60" HorizontalAlignment="Center" HorizontalContentAlignment="Stretch" VerticalContentAlignment="Stretch">
+                    <Grid HorizontalAlignment="Stretch" VerticalAlignment="Stretch" Width="Auto" Height="Auto">
+                        <SymbolIcon Symbol="Sync" HorizontalAlignment="Center"/>
+                        <InfoBadge Background="#C42B1C" HorizontalAlignment="Right" VerticalAlignment="Top">
+                            <InfoBadge.IconSource>
+                                <FontIconSource FontFamily={StaticResource SymbolThemeFontFamily} Glyph="&#xF13C;"/>
```
Comment (niels9001) [inline WinUIGallery/ControlPages/InfoBadgePage.xaml:107]:
```suggestion
                                <FontIconSource FontFamily="{StaticResource SymbolThemeFontFamily}" Glyph="&#xF13C;"/>
```

**Review by niels9001 — CHANGES_REQUESTED** [review]
@ghost1372 Except for two missing quotes this seems to work well :)! Thanks for adding!

**Review by niels9001 — APPROVED** [review]

File: WinUIGallery/ControlPages/InfoBadgePage.xaml (hunk 4 above)
Comment (ghost1372) [reply WinUIGallery/ControlPages/InfoBadgePage.xaml]: tnx

**Pull request merged**